app/javascript/components/upload/upload-utils.js

```diff
@@ -249,8 +249,8 @@ const plainTextExtensions = ['.txt', '.tsv', '.text', '.csv']
 const mtxExtensions = ['.mtx', '.mm', '.txt', '.text']
 const imageExtensions = ['.jpeg', '.jpg', '.png', '.bmp']
 const miscExtensions = ['.txt', '.text', '.tsv', '.csv', '.jpg', '.jpeg', '.png', '.pdf',
-  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.h5', '.h5ad', '.h5an',
-  '.ipynb', '.Rda', '.rda', '.Rds', '.rds']
+  '.doc', '.docx', '.xls', '.xlsx', '.ppt', '.pptx', '.zip', '.loom', '.h5an',
+  '.ipynb', '.Rda', '.rda']
```

Comment:
Blocker: .Rda and .rda were both added for Seurat support, simultaneously with .Rds and .rds.  For consistency, we should remove the former here since we're removing the latter.

Context:
* Original user request: https://broadinstitute.zendesk.com/agent/tickets/143271
* Related PR: https://github.com/broadinstitute/single_cell_portal_core/pull/1002

Reply:
I presume then `.rda` and `.Rda` ought to be added to the Seurat extension list?
Also thank you for the context super helpful 👍

Reply:
oops just saw your other comment got it will add to the Seurat list